dashboard/app/views/shared/_district_dropdown.haml

```diff
@@ -10,10 +10,10 @@
       .question Select school type
       %select#school-type.form-control{name: "#{form_name}[school_type]", type: "select", required: false}
         %option{value: "", selected: true, disabled: true}
-        %option{value: "charter"} Charter
-        %option{value: "private"} Private
-        %option{value: "public"} Public
-        %option{value: "other"} Other
+        %option{value: DistrictDropdownConstants::SCHOOL_TYPE_CHARTER} Charter
+        %option{value: DistrictDropdownConstants::SCHOOL_TYPE_PRIVATE} Private
+        %option{value: DistrictDropdownConstants::SCHOOL_TYPE_PUBLIC} Public
+        %option{value: DistrictDropdownConstants::SCHOOL_TYPE_OTHER} Other
```

Comment:
Good call. You can follow [this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/pd/workshop.rb#L33) pattern.